src/utils/formatting.ts

```diff
@@ -110,3 +112,27 @@ export const formatSeriesMetaString = (seasonNumber?: string, episodeNumber?: st
 
   return seasonNumber && seasonNumber !== '0' ? `S${seasonNumber}:E${episodeNumber}` : `E${episodeNumber}`;
 };
+
+export const formatLiveEventMetaString = (media: PlaylistItem) => {
+  const metaData = [];
+  const scheduled = formatVideoSchedule(media.scheduledStart, media.scheduledEnd);
+
+  if (scheduled) metaData.push(scheduled);
+  if (media.duration) metaData.push(formatDuration(media.duration));
+  if (media.genre) metaData.push(media.genre);
+  if (media.rating) metaData.push(media.rating);
+
+  return metaData.join(' • ');
+};
+
+export const formatVideoSchedule = (scheduledStart?: Date, scheduledEnd?: Date) => {
+  if (!scheduledStart) {
+    return '';
+  }
+
+  if (!scheduledEnd) {
+    return format(scheduledStart, 'PPP • p');
+  }
+
+  return `${format(scheduledStart, 'PPP • p')} - ${format(scheduledEnd, 'p')}`;
```

Comment:
I think we should use the `Intl.DateTimeFormat` here instead of `date-fns` i18n functions. Because we don't know which languages are supported and this prevents publishers from adding the date fns `locale` file.

```ts
const formatLocalizedDate = (date: Date, locale: string) => new Intl.DateTimeFormat(locale, { day: 'numeric', month: 'numeric', year: 'numeric' }).format(date);

const formatLocalizedTime = (date: Date, locale: string) => new Intl.DateTimeFormat(locale, { hours: 'numeric', minutes: 'numeric', seconds: 'numeric' }).format(date);

const formatLocalizedDateTime = (date: Date, locale: string, separator = ' ') => `${formatLocalizedDate(date, locale)}${separator}${formatLocalizedTime(date, locale)}`;
```

```suggestion
  if (!scheduledEnd) {
    return formatLocalizedDateTime(scheduledStart, locale, '•');
  }

  return `${formatLocalizedDateTime(scheduledStart, locale, '•')} - ${formatLocalizedTime(scheduledEnd, 'locale')}`;
```